tools/finish_release.py

```diff
@@ -68,7 +64,7 @@ def parse_args(args):
     # Use the file's docstring for the help text and don't let argparse reformat it.
     parser = argparse.ArgumentParser(description=__doc__,
                                      formatter_class=argparse.RawDescriptionHelpFormatter)
-    parser.add_argument('githubpat', help='path to your GitHub personal access token')
+    parser.add_argument('--css', type=str, help='hostname of code signing server')
```

Comment:
Making this no longer required like you did is probably the correct choice so the script works without `--css` with `--snaps-only`, but can we add logic somewhere to fail fast if `parsed_args.publish_windows` is `True` and `--css` is not set? I think we could either do it at the bottom of this function or in `main`.

Reply:
Alternatively, you could just make this parameter required again.